src/extension.ts

```diff
@@ -293,6 +293,13 @@ export function activate(context: vscode.ExtensionContext) {
     vscode.languages.registerDefinitionProvider(md, new MarkdownDefinitionProvider())
   );
 
+  let findBacklinksDisposable = vscode.commands.registerCommand('vscodeMarkdownNotes.findBacklinks', () => {
+    const filename = vscode.window.activeTextEditor?.document.fileName
+    if (filename !== undefined)
+      vscode.commands.executeCommand("workbench.action.findInFiles", { query: `[[${basename(filename)}]]`, triggerSearch: true, filesToInclude: '*.md, *.markdown' })
```

Comment:
This is cool, I didn't know about this API.

I definitely think we should add something that opens a search for `#tag` with a keyboard shortcut when your cursor is over the word `#tag` (or when you click on `#tag`)